#! /bin/sh /usr/share/dpatch/dpatch-run
## gcc4.3-includes.dpatch by Sebastian Harl <sh@tokkee.org>
##
## DP: This patch adds missing includes required by gcc-4.3.

@DPATCH@

--- a/include/vigra/imageiteratoradapter.hxx	2007-01-27 05:54:46.000000000 +0100
+++ b/include/vigra/imageiteratoradapter.hxx	2007-12-27 18:15:52.000000000 +0100
@@ -38,6 +38,8 @@
 #ifndef VIGRA_IMAGEITERATORADAPTER_HXX
 #define VIGRA_IMAGEITERATORADAPTER_HXX
 
+#include <cstdlib>
+
 #include <iterator>   // iterator tags
 
 namespace vigra {
--- a/src/gpu.cc	2006-12-08 09:40:58.000000000 +0100
+++ b/src/gpu.cc	2007-12-28 04:04:49.000000000 +0100
@@ -18,6 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  */
 
+#include <cstdlib>
+
 #include <iostream>
 
 #include "gpu.h"
